libs/sdk-core/src/models.rs

```diff
@@ -1584,6 +1584,127 @@ impl PaymentPathEdge {
     }
 }
 
+pub(crate) mod sanitize {
+    use anyhow::Result;
+    use serde::Serialize;
+
+    use crate::{
+        FullReverseSwapInfo, OpeningFeeParams, ReverseSwapInfoCached, SwapInfo, SwapStatus,
+    };
+
+    pub(crate) trait Sanitize<R, S> {
```

Comment:
Not sure if we need a trait here. I think we could save some code if we only implement the `From` trait for the FullReverseSwapInfoSanitized from FullReverseSwapInfo ?